### ISSUE TITLE
Web: Have search textbox pull focus in emoji picker after opening

### ISSUE THREAD

**Issue by Juicysteak117** [bluesky-social/social-app#1328]
Currently, when opening the emoji picker on web, the cursor stays within the post composer. It would be good to pull focus into the emoji search box when the emoji picker opens given that desired behavior after opening is to immediately search for the desired emoji in most situations.

**Comment by pfrazee** [comment]
@ansh we should also not move the cursor for the insert

**Comment by Juicysteak117** [comment]
Yeah currently the text cursor gets lost entirely after inserting so I wasn't sure if I should make that a separate issue but it's related.

**Comment by pfrazee** [comment]
@Juicysteak117 Yeah that's related to the issue you're reporting; I'm tacking on an additional ask, which is that the emoji should be inserted where the cursor currently is
